packages/components/src/public/QueryInfo.ts

```diff
@@ -10,6 +10,7 @@ import { insertColumnFilter, QueryColumn } from './QueryColumn';
 import { SchemaQuery } from './SchemaQuery';
 import { QuerySort } from './QuerySort';
 import { naturalSortByProperty } from './sort';
+import { IDENTIFYING_FIELDS_VIEW_NAME } from './QueryModel/SaveViewModal';
```

Comment:
nit: Doesn't look like it was added with this commit but I recommend we co-locate this constant with similar constants declared on `ViewInfo`.

Reply:
Done.